src/docs/rtc/README.md

```diff
@@ -38,3 +38,17 @@ Consulta Estado Envío de AEC al Registro; Consulta de Cesiones",
 - Nombre original: `RTCDocum.pdf`
 - MD5: `c8b971e86ba7dba5616f8b7f5936933c`
 - SHA256: `d4d30f7e6e844aa7a4ce128e056322de1baef061d4a10ea0a5ddad5b2ad02fb6`
+
+
+### Envío automático de Archivos Electrónicos de Cesión (AEC)
+
+"Manual de Desarrollador Externo" - "Envío Automático de Archivos Electrónicos de Cesión
+(AEC)",
+[v1.0 (2007-01-31)](RPETCEnvio/2007-01-31-envío-automático-de-archivos-electrónicos-de-cesión-v1.0.pdf)
```

Comment:
PDF link is broken. Please remove the non-English characters.

Reply:
Good catch! I forgot to update the README